spec/controllers/v2/concerns/rendering_spec.rb

```diff
@@ -50,7 +50,9 @@
     expect(response_body_data).to match_array(collection)
 
     response_body_data.each do |item|
-      expect(item.keys.count).to eq(attributes.keys.count + 2)
+      default_attributes = %i[id type].freeze
+      expected_attributes = default_attributes + attributes.keys
+      expect(item.keys.count).to eq(expected_attributes.count)
```

Comment:
I am not saying this is wrong, just need some explanation for this

Reply:
It could be simplified to

```suggestion
      expect(item.keys.count).to eq((attributes.keys + %i[id type]).count)
```

My main focus is to get rid of the *magic* number `2` and clarify what it stands for.

Reply:
This is outside of the scope of this PR

Reply:
Ack